client/my-sites/customer-home/components/celebrate-launch-modal.scss

```diff
@@ -122,3 +136,7 @@
 		text-decoration: underline;
 	}
 }
+
+.tooltip.popover {
```

Comment:
Hi! I think you should scope this CSS to celebration modal somehow, or otherwise the CSS leaks to _all_ tooltips in rest of Calypso as the CSS is in global scope. Alternatively make the change directly in "tooltip" component.

Reply:
Bringing this to the attention of cylon 👍